From a8ccd8401571e7455e98370635349a0b93238ea5 Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Wed, 5 Dec 2018 11:28:14 +0200
Subject: [PATCH 291/741] net/phy: xgkr: Local/Remote training functions
 renaming

commit 9ce16e65ed25a5bf4a6abf5391a66371563225c0 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Intuitive functions renaming to reflect exactly their purpose: Local transmitter training / Remote transmitter training.

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/phy/fsl_backplane.c | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/drivers/net/phy/fsl_backplane.c b/drivers/net/phy/fsl_backplane.c
index 68ea2135c817..d9544ead8df8 100644
--- a/drivers/net/phy/fsl_backplane.c
+++ b/drivers/net/phy/fsl_backplane.c
@@ -517,7 +517,7 @@ static void initialize(struct xgkr_params *xgkr)
 	ld_coe_status(xgkr);
 }
 
-static void train_tx(struct xgkr_params *xgkr)
+static void train_remote_tx(struct xgkr_params *xgkr)
 {
 	struct tx_condition *tx_c = &xgkr->tx_c;
 	bool bin_m1_early, bin_long_early;
@@ -1034,7 +1034,7 @@ static void preset(struct xgkr_params *xgkr)
 	ld_coe_status(xgkr);
 }
 
-static void train_rx(struct xgkr_params *xgkr)
+static void train_local_tx(struct xgkr_params *xgkr)
 {
 	int request, old_ld_status;
 
@@ -1101,7 +1101,7 @@ static void xgkr_start_train(struct xgkr_params *xgkr)
 	int val = 0, i;
 	int lt_state;
 	unsigned long dead_line;
-	int rx_ok, tx_ok;
+	int lp_rx_ready, tx_training_complete;
 	u32 lt_timeout = 500;
 
 	init_xgkr(xgkr, 0);
@@ -1147,8 +1147,8 @@ static void xgkr_start_train(struct xgkr_params *xgkr)
 		}
 
 		/* init process */
-		rx_ok = false;
-		tx_ok = false;
+		lp_rx_ready = false;
+		tx_training_complete = false;
 		/* the LT should be finished in 500ms, failed or OK. */
 		dead_line = jiffies + msecs_to_jiffies(lt_timeout);
 
@@ -1171,17 +1171,17 @@ static void xgkr_start_train(struct xgkr_params *xgkr)
 				break;
 			}
 
-			rx_ok = check_rx(xgkr);
-			tx_ok = tx_c->tx_complete;
+			lp_rx_ready = check_rx(xgkr);
+			tx_training_complete = tx_c->tx_complete;
 
-			if (rx_ok && tx_ok)
+			if (lp_rx_ready && tx_training_complete)
 				break;
 
-			if (!rx_ok)
-				train_rx(xgkr);
+			if (!lp_rx_ready)
+				train_local_tx(xgkr);
 
-			if (!tx_ok)
-				train_tx(xgkr);
+			if (!tx_training_complete)
+				train_remote_tx(xgkr);
 
 			usleep_range(100, 500);
 		}
-- 
2.17.1

